src/components/column-sorting/column-sorting.component.less

```diff
@@ -0,0 +1,36 @@
+@import "../../styles/variables.less";
+ux-column-sorting{
+    .ux-column-sorting {
+        padding-left: 5px;
+        display: inline-block;
+        width: 36px;
+        height: 28px;
```

Comment:
Is that expected? I thought that was a bug in the angular 1 version? Looking at an old version they are inline:

![image](https://cloud.githubusercontent.com/assets/20795331/26101897/afb11dc0-3a2a-11e7-8ba5-9c2c15f0ab24.png)